dashboard/test/ui/features/project_gallery.feature

```diff
@@ -0,0 +1,45 @@
+# TODO (Erin B.) remove ?showFeatured=1 param from project gallery url during flag removal.
+
+@dashboard_db_access
+Feature: Public Project Gallery
+
+  Background:
+    Given I create a teacher named "Project_Czar"
+    And I give user "Project_Czar" project validator permission
+
+Scenario: Published Projects Show In Recency Order
+  Then I make a playlab project named "Older Published"
+  Then I publish the project
+  Given I am on "http://studio.code.org/projects/public"
+  Then I wait until element ".project_card" is in the DOM
+  Then I wait until element "#ui-project-name" is in the DOM
+  Then I wait until the first "#ui-project-name" contains text "Older Published"
```

Comment:
FYI: This new step "I wait until the first" is not strictly necessary, because you can use `I wait until "#ui-project-name:nth(0)" ...` instead. However I think this new step actually improves the readability nicely, so I'm in favor of keeping it. Let's just keep `nth` in mind for other uses in the future. e.g.: https://github.com/code-dot-org/code-dot-org/blob/beb45173d2b124743ef55fb1370867ae2225b363/dashboard/test/ui/features/teacher_dashboard.feature#L37